var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -61,6 +61,14 @@ class Cp2k(MakefilePackage, CudaPackage):
     variant('cuda_blas', default=False,
             description=('Use CUBLAS for general matrix operations in DBCSR'))
 
+    HFX_LMAX_RANGE = range(4, 8)
+
+    variant('lmax',
+            description='Maximum supported angular momentum (HFX and others)',
+            default='5',
+            values=tuple(str(i) for i in HFX_LMAX_RANGE),
```

Comment:
You should be able to use ints here, just do `list(HFX_LMAX_RANGE)`.

Reply:
done, thanks!